_build/default/lib/beam_stats/src

```diff
@@ -0,0 +1 @@
+../../../../src
```

Comment:
Nothing in `_build` should ever be committed. The project currently uses rebar2, so a rebar3 conversion should first be done, which will include adding `_build/` to `.gitignore`.

Reply:
Right, that's what I usually do! I actually compiled everything with rebar3, but I didn't edit  `.gitignore`.